server/helpers/UserHelper.js

```diff
@@ -0,0 +1,35 @@
+import { User } from '../sequelize/models/index';
+
+/**
+ * This class contains all methods
+ * required to handle
+ * user data to the database.
```

Comment:
Consider refactoring the comment ...**user data modifications in the database**.

Reply:
done